### PR TITLE
Vnet vxlan extra RIF enhancement

### PR DESCRIPTION
**Pull request by theasianpianist**
### Description of PR

Summary:
Allow the VNET VxLAN test to be run with more than one RIF (VLAN interface) per VNET.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
The existing VNET VxLAN test only allows for 1 VLAN interface per VNET. Adding this enhancement allows for testing of more scenarios.
#### How did you do it?
* Add `num_intf_per_vnet` CLI parameter (defaults to 1)
* Modify `vnet_config.j2` to create additional VLAN interfaces starting at VLAN ID 2000
	* The first VLAN interface in each VNET is still created in the range starting at 3001 and should not be affected by this change.
    * All interfaces in the same VNET share the same port.
    * E.g. for 4 VLAN interfaces per VNET, VNET 1 will contain VLAN interfaces 3001, 2000, 2001, and 2002 on Ethernet28. VNET 2 will contain 3002, 2003, 2004, and 2005 on Ethernet32
* In Serv2Serv scenario in the PTF test, look for other VLANs in the same VNET and send packet to each to verify connectivity.
* Send ARP replies to all VLANs on a given port in `arp_responder.py`

#### How did you verify/test it?
* Run the test with default parameters (1 VLAN interface per VNET) and verify all scenarios pass
* Run the test with more than 1 interface per VNET and verify all scenarios pass
### Documentation 

`./run_tests.sh -n <testbed> -i <inventory> -u -c vxlan/test_vnet_vxlan.py -e --disable_loganalyzer -e --num_intf_per_vnet=<x>`

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

**Comment by prsunny** [comment]
Waiting for test result from @volodymyrsamotiy before merge

File: tests/vxlan/__init__.py
Comment (prsunny) [inline tests/vxlan/__init__.py:36]: Why is this default values changed?

Comment (theasianpianist) [reply tests/vxlan/__init__.py:36]: The parameters `num_vnet` and `num_intf_per_vnet` required a pytest default parameter to be able to print a warning if the number of RIFs exceeded 128. I added defaults for the other parameters (which match the default values given in the `vnet_config.j2` template) for consistency.

Comment (theasianpianist) [reply tests/vxlan/__init__.py:36]: The warning is [here](https://github.com/Azure/sonic-mgmt/pull/2236/files#diff-8399274c72942792acb3c281a7db7c9bR140) and the default values in the J2 template are defined around [here](https://github.com/Azure/sonic-mgmt/blob/d190bb2a5582ec3fda3aa2644c87e84f0c0fbab4/tests/vxlan/templates/vnet_config.j2#L41)

**Review by volodymyrsamotiy — APPROVED** [review]
Tried to run the test with the "num_intf_per_vnet" argument and it passed.
```
==================================== test session starts ====================================
platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
ansible: 2.8.7
rootdir: /auto/sw/work/sonic/volodymyrs/sonic-mgmt_local/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collected 3 items

vxlan/test_vnet_vxlan.py::test_vnet_vxlan[Disabled] PASSED                            [ 33%]
vxlan/test_vnet_vxlan.py::test_vnet_vxlan[Enabled] PASSED                             [ 66%]
vxlan/test_vnet_vxlan.py::test_vnet_vxlan[Cleanup] PASSED                             [100%]

================================ 3 passed in 183.20 seconds =================================
```

**Pull request merged**
